R/returning.R

```diff
@@ -59,3 +59,13 @@ porcelain_returning_binary <- function(status_code = 200L) {
   process <- identity
   porcelain_returning(content_type, process, validate, status_code)
 }
+
+
+##' @export
+##' @rdname porcelain_returning
+porcelain_returning_text <- function(status_code = 200L) {
```

Comment:
how does this relate to `accept` headers? can the same endpoint be distinguished just by different accept headers, or does an endpint have to either return `json` or `text` regardless

Reply:
we can't actually respond to accept headers in porcelain (I thought I'd made a comment here, but it is there in the orderly.server code that's not pushed yet). We should do, but that's a much bigger change, I think